packages/admin-ui/ui/src/utils/get-admin-path.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * returns the full admin path using the window object
+ * @returns {string} - admin path
+ */
+export function getFullAdminPath() {
```

Comment:
Suggestion: getAdminPath? Don't seem that we ever get non full path no?

Reply:
Admin path in the context of our plugin is whatever is provided in the `path` config in `medusa-config.js`. With window.location.origin, this becomes a "full" admin root path.

Reply:
I think we cant keep it like it 👍